docs/example-solutions/examples.rst

```diff
@@ -19,7 +20,7 @@ of hyperparameters:
    cd mnist_pytorch
    det experiment create const.yaml .
 
-For an introduction to using the training API, please see the Training APIs section.
+For an introduction to using the training API, please visit the Training APIs section.
```

Comment:
Can we create a link to the "Training APIs section"?

Reply:
good point, will fix this in the new docusaurus site once new sections exist
https://determinedai.atlassian.net/jira/software/projects/DOCS/boards/41?selectedIssue=DOCS-16